code/src/components/MessageInput.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+
+export const MessageInput = ({ getMessages }) => {
+
+  const MESSAGE_URL = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+
+  const [message, setMessage] = useState("")
+
+  const postMessage = message => {
+    fetch(MESSAGE_URL, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ message: message })
```

Comment:
Since both key and value inside this object have same name, we can simplify this :

```
body: JSON.stringify({ message })
```